### ISSUE TITLE
When a job is deleted by SGE  on Rye an error message is not evoked

### ISSUE THREAD

**Issue by vikramalva** [proteinevolution/Toolkit#619]
This worked all the time in the past, but this must have gone broken because of recent changes. Works on Olt.

**Comment by zy4** [comment]
I don't know what kind of error message you are referring to. Is it something the user sees in the frontend?

**Comment by vikramalva** [comment]
Yes. I started three jobs yesterday and killed them using qdel, but they are still shown to be running.

![screenshot-2018-7-22 hhpred bioinformatics toolkit](https://user-images.githubusercontent.com/20257466/43043653-7e91a468-8d98-11e8-82bc-b26dbd71d6a9.png)
